pyproject.toml

```diff
@@ -55,6 +55,17 @@ cov = [
   "cov-report",
 ]
 
+[tool.hatch.envs.hatch-test]
+dependencies = [
+  "coverage[toml]>=6.5; python_version >= '3.8'",
+  "pytest",
+  "flask",
+  "flask-jwt-extended",
+]
+
+[[tool.hatch.envs.hatch-test.matrix]]
+python = ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
i thought 3.7 does not support hatch test?

Reply:
Yes if we install hatch under py=3.7, it will install an early version which is like 1.7.0. There are a lot of hatch changes that won't work in this early version. That's another reason why we may consider deprecating docker-hatch some day.